docs/CONTRIBUTING.md

```diff
@@ -111,73 +120,88 @@ yarn test:standalone-release
 yarn package
 ```
 
-NOTE: On Linux, the currently running distro will become the minimum supported version.
-In our GitHub Actions CI, we use CentOS 7 for maximum compatibility.
-If you need your builds to support older distros, run the build commands
-inside a Docker container with all the build requirements installed.
+> On Linux, the currently running distro will become the minimum supported
+> version. In our GitHub Actions CI, we use CentOS 7 for maximum compatibility.
+> If you need your builds to support older distros, run the build commands
+> inside a Docker container with all the build requirements installed.
 
-## Testing
+### Test
 
 There are three kinds of tests in code-server:
 
-1. unit tests
-2. integration tests
-3. end-to-end tests
+1. Unit tests
+2. Integration tests
+3. End-to-end tests
 
-### Unit Tests
+### Unit tests
```

Comment:
Ah I see you're only capitalizing the first word. I think either works as long as we stay consistent